dashboard/app/models/pd/teachercon_survey.rb

```diff
@@ -81,14 +89,19 @@ def self.required_fields
       :facilitators_could_improve,
       :liked_most,
       :would_change,
-      :give_permission_to_quote,
 
       :instruction_focus,
       :teacher_responsibility,
       :teacher_time,
     ].freeze
   end
 
+  def self.required_fields
+    public_required_fields + [
+      :give_permission_to_quote
+    ].freeze
```

Comment:
AFAIK, the `freeze` applies to the temporary one element array containing `:give_permission_to_quote`. If the `freeze` is meant to apply to the return, parens are necessary. Otherwise, I don't see the purpose of the `freeze`.

Also elsewhere throughout this PR.